hack/prune-api-description-whitelist.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
I don't think I care about this script.  I'm inclined to make people keep up with it manually.  You have good messages in the verify script already.